packages/airnode-validator/src/config/config.ts

```diff
@@ -440,3 +446,4 @@ export type Trigger = SchemaType<typeof triggerSchema>;
 export type Triggers = SchemaType<typeof triggersSchema>;
 export type Heartbeat = SchemaType<typeof heartbeatSchema>;
 export type Amount = SchemaType<typeof amountSchema>;
+export type EnabledGatewaySchema = SchemaType<typeof enabledGatewaySchema>;
```

Comment:
```suggestion
export type EnabledGateway = SchemaType<typeof enabledGatewaySchema>;
```
This is a great idea (and I will make sure to add to cloud handlers as well), but perhaps rename to without the `Schema` part for the type name to keep consistent with the other types.